### PR TITLE
Doc: Readme fixes and update to v6.0.1

### PR DESCRIPTION
**Pull request by avirtopeanu-ionos**
## What does this fix or implement?

Currently there are some problems with the Readme.md file:

In the example code, import and all objects of type `DataCenterApi` need to be renamed to `DataCentersApi`

For pom.xml and gradle, versions need to have "v" pre-pended: example for pom.xml `<version>6.0.0</version>` into `<version>v6.0.0</version>`

Also, starting with v6.0.1, the method `DataCentersApi::datacentersGet` will need to have 3 more parameters:  String orderBy, Integer maxResults, Map<String, String> filters. This needs to be reflected in the example code too

## Checklist
- [x] PR name added as appropriate (e.g. feat/fix/doc/test/refactor/etc)
- [ ] Tests added or updated
- [x] Documentation updated
- [ ] Changelog updated and version incremented (label: upcoming release)
- [ ] Github Issue linked if any
- [x] Jira task updated

### CODE REVIEW

**Review by iblindu — APPROVED** [review]
:heart_eyes:

**Review by rmocanu-ionos — APPROVED** [review]

**Review by mimihalescu — APPROVED** [review]
Great job! 🥇

**Comment by avirtopeanu-ionos** [comment]
Sorry guys, used the old team for seeing who I needed to add as reviewers and added too many

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Add this dependency to your project's POM:
 <dependency>
   <groupId>com.ionoscloud</groupId>
   <artifactId>ionos-cloud-sdk</artifactId>
-  <version>6.0.0</version>
+  <version>v6.0.1</version>
```
Comment (cristiGuranIonos) [inline README.md:42]: Good find

**Pull request merged**